docs/installation/server-config/ccloud-ksql.rst

```diff
@@ -0,0 +1,45 @@
+.. _install_ksql-ccloud:
+
+Using KSQL with |ccloud|
+========================
+
+You can connect KSQL to your |cp| Kafka cluster in |ccloud|.
+
+**Prerequisites**
+
+- :ref:`Confluent Platform <installing_cp>`
+
+#.  Customize your ``/etc/ksql/ksql-server.properties`` properties file.
+
+    .. code:: bash
+
+        # a comma separated list of the the ccloud broker endpoints.
+        # eg. r0.great-app.confluent.aws.prod.cloud:9092,r1.great-app.confluent.aws.prod.cloud:9093,r2.great-app.confluent.aws.prod.cloud:9094
+        bootstrap.servers=<broker-endpoint1, broker-endpoint2, broker-endpoint3>
+        #
+        ksql.sink.replicas=3
+        #
+        replication.factor=3
+        #
+        security.protocol=SASL_SSL
+        #
+        sasl.mechanism=PLAIN
+        #
+        sasl.jaas.config=\
+            org.apache.kafka.common.security.plain.PlainLoginModule required \
+            username="<confluent cloud access key>" \
+            password="<confluent cloud secret>";
+        # Recommended performance settings
+        producer.retries=2147483647
+        producer.confluent.batch.expiry.ms=9223372036854775807
+        producer.request.timeout.ms=300000
+        producer.max.block.ms=9223372036854775807
+
+#.  Start or restart |cp|.
+
+    .. code::
+
+        $ <path-to-confluent>/confluent start
```

Comment:
It might may more sense to say 'restart the ksql server', and may be link to the `confluent restart` command or the `ksql-server-stop && ksql-server-start`. Sandbox environments will use the former while production environments will use the latter.